.travis.yml

```diff
@@ -20,4 +20,5 @@ branches:
     - master
 
 script:
-  - make style codecov
+  - go get -v github.com/Masterminds/glide
```

Comment:
This should go in `before_install`, `install` or `before_script` so if this step fails, the build errors out instead of failing.

Reply:
+1

Never install external tools during a build.  Use the installation step(s) in `.travis.yml`.  This gives you a clearly understandable failure log.

Reply:
Sounds good. I'll rebase this PR on top of master where I made that fix already.